l5kit/l5kit/dataset/ego.py

```diff
@@ -21,6 +21,7 @@ def __init__(
             zarr_dataset: ChunkedDataset,
             rasterizer: Rasterizer,
             perturbation: Optional[Perturbation] = None,
+            render_path_prior: bool = False,
```

Comment:
Do we need to add render_path_prior at Line 168?: https://github.com/lyft/l5kit/pull/248/files#diff-e40a85c9573b89ac8b28c54ddf84835cad03fdcc9760a35762cd95ffff7d464bL168

Reply:
ah yeah that's a good point! I really don't like passing this arg...another option would be to add it in the `cfg`